.zenodo.json

```
@@ -0,0 +1,59 @@
+{
+    "creators":[
+        {
+            "name": "SuperDARN Data Analysis Working Group"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Billett, D.D.",
+            "orcid": "0000-0002-8905-8609"
+        },
+        {
+            "affiliation": "The Unviersity Centre in Svalbard",
+            "name": "Bland, Emma",
+            "orcid": "0000-0002-0252-0400"
+        },
+        {
+            "affiliation": "Naval Research Laboratory",
+            "name": "Burrell, A.G.",
+            "orcid": "0000-0001-8875-9326"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Kotyk, K."
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Ponomarenko, P.V.",
+            "orcid": "0000-0001-8407-0193"
+        },
+        {
+            "affiliation": "SRI International",
+            "name": "Reimer, A.S.",
+            "orcid": "0000-0002-4621-3453"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Schmidt, M."
```

``` suggestion
            "name": "Schmidt, M."
            "orchid": "0000-0002-3265-977X"
```
Surprisingly enough I am a scientist that has been on 4 papers :p

---

Sorry I wasn't able to find yours!  I think I meant to put that in the notes.  If I didn't find someone's ORCID it's not because I didn't think it was important, I just couldn't find it!

I'll see if I can fix the typo real quick/slick with git reviewing.
```suggestion
            "name": "Schmidt, M."
            "orcid": "0000-0002-3265-977X"
```

---

``` suggestion
            "name": "Schmidt, M.T"
            "orcid": "0000-0002-3265-977X"
```

How about this? Realized M. Schmidt is probably very common

---

Whoops, I'd missed this part here of adding an initial.  Though you'll want to add a period there I'd think.  Sorry I'd missed the orcid, just didn't come up quickly/easily in the database.  
```suggestion
            "name": "Schmidt, M. T."
```